test/e2e/discovery/service_discovery.go

```diff
@@ -238,47 +251,79 @@ func RunServicesPodAvailabilityMutliClusterTest(f *lhframework.Framework) {
 		f.AwaitServiceImportIP(framework.ClusterA, nginxServiceClusterA)
 	}
 
-	verifyServiceIpWithDig(f.Framework, framework.ClusterA, nginxServiceClusterA, netshootPodList, checkedDomains, true)
-	verifyServiceIpWithDig(f.Framework, framework.ClusterA, nginxServiceClusterB, netshootPodList, checkedDomains, true)
+	clusterBDomain := getClusterDomain(f.Framework, framework.ClusterB, netshootPodListB)
+
+	By(fmt.Sprintf("Fetching nginxServiceClusterB from %q, by overriding with %q", clusterBName, clusterBDomain))
```

Comment:
I'm not clear on this output - it seems it should be after L258. But do we really need this extra output in this one case?

Reply:
Removed